tdrs-frontend/src/assets/Button.scss

```diff
@@ -0,0 +1,5 @@
+.usa-button,
+.usa-button:hover {
+  font-size: 1rem;
+  background-color: $primary-medium-blue;
+}
```

Comment:
I was individually styling multiple buttons the same way so I pulled that out into a single button style sheet.